.travis.yml

```diff
@@ -1,12 +1,21 @@
 sudo: false
+
 language: rust
-rust: stable
```

Comment:
lets keep stable, to make sure that we don’t start using nightly only features.

checkout https://dev.to/cad97/great-rust-ci-1fk6 for a nice setup description